Packs/FTP/Integrations/FTP/FTP.yml

```diff
@@ -0,0 +1,62 @@
+category: Utilities
+commonfields:
+  id: FTP
+  version: -1
+configuration:
+- display: 'Host'
+  name: host
+  required: true
+  section: Connect
+  type: 0
+- display: 'Port'
+  name: port
+  defaultvalue: "21"
+  required: false
+  section: Connect
+  type: 0
+- display: username
```

Comment:
```suggestion
- display: Username
```

Reply:
@RotemAmit I just pushed one commit. Please take a look at it. Thanks.